README.md

```diff
@@ -322,7 +322,7 @@ A complete list of supported parameters for the `runs download` command is avail
 This command retrieves details of properties in a namespace.
 
 Properties in a namespace can be filtered out by using `--prefix`, `--infix` and/or `--suffix`, or `--name`.
-Currently one format is supported: "summary".
+Two formats are supported: 'summary', 'raw'
```

Comment:
The default is summary.

Reply:
Perhaps adding an example of the output with the raw format for two properties ?